src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -1591,19 +2099,33 @@ public enum RisingRivals implements LogicCardInfo {
           weakness R
           resistance P, MINUS20
           move "Shell Scatter", {
-            text "can’t use Shell Scatter during your next turn."
+            text "This attack does 10 damage to each of your opponent's Benched Pokémon. Foretress G can’t use Shell Scatter during your next turn."
             energyCost C
-            attackRequirement {}
+            attackRequirement {
+              assert opp.bench : "Your opponent has no Benched Pokémon"
+            }
             onAttack {
-              damage 0
+              opp.bench.each {
```

Comment:
Does not say Benched in the actual card text.

![](https://pkmncards.com/wp-content/uploads/forretress-g-rising-rivals-rr-61.jpg)